examples/notebooks/PII/Run_your_first_PII_redactor_transform.ipynb

```diff
@@ -0,0 +1,579 @@
+    "\n",
+    "- **Creating the Final Output:** After the redaction process, a new output Parquet file is generated in **output-redacted** folder, containing the same structured data as the original but with all sensitive details securely masked to prevent unauthorized access or exposure.\n",
+    "\n",
+    "\n",
+    "### Why is PII Redaction Important?\n",
+    "\n",
+    " **Data Privacy Compliance**: Adheres to regulations like GDPR, HIPAA, or CCPA that mandate safeguarding customer information.\n",
+    "\n",
+    " **Risk Mitigation**: Prevents unauthorized access to or misuse of sensitive data.\n",
+    "\n",
+    " **Automation Benefits**: Simplifies and accelerates the process of securing information in large-scale document handling.\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "10a3ed52-68aa-45bb-a2d3-1b08e5ad2ab2",
+   "metadata": {},
+   "source": [
+    "### Pre-req: Install data-prep-kit dependencies"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "fd7131c9-af67-4ac5-9d3d-a90ab7d11898",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%%capture logpip --no-stderr\n",
+    "!pip install data-prep-toolkit==0.2.2\n",
+    "!pip install 'data-prep-toolkit-transforms[pii_redactor]==0.2.2'\n",
+    "!pip install 'data-prep-toolkit-transforms[pdf2parquet]==0.2.2'\n",
+    "!pip install pdfplumber\n",
+    "!pip install spacy"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "b3cfa3c3-6619-4cd4-887f-6d269f896210",
+   "metadata": {},
+   "source": [
+    "###  Figure out Runtime Environment"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "id": "453b6ba9-253a-47c7-9b6c-999d01734114",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "NOT in Colab\n"
+     ]
+    }
+   ],
+   "source": [
+    "import os\n",
+    "\n",
+    "if os.getenv(\"COLAB_RELEASE_TAG\"):\n",
+    "   print(\"Running in Colab\")\n",
+    "   RUNNING_IN_COLAB = True\n",
+    "else:\n",
+    "   print(\"NOT in Colab\")\n",
+    "   RUNNING_IN_COLAB = False"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "9cab116e-b772-4e4b-ab2e-d22593b26a7a",
+   "metadata": {},
+   "source": [
+    "### Download Data if running on Google Colab"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "id": "7e6d4c4d-990a-4938-b39a-ba761a638931",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "if RUNNING_IN_COLAB:\n",
+    "  !mkdir -p 'input-data'\n",
+    "  !wget -O 'input-data/Invoice.pdf' 'https://raw.githubusercontent.com/PoojaHolkar/data-prep-kit/refs/heads/dev/examples/notebooks/PII/invoicedata/Invoice.pdf'\n",
```

Comment:
Please change .../examples/notebooks/PII/invoicedata/Invoice.pdf to .../examples/notebooks/PII/input-data/Invoice.pdf